params/config.go

```diff
@@ -567,10 +570,6 @@ func (c *ClusterConfig) Validate(validate *validator.Validate) error {
 		return err
 	}
 
-	if c.Fleet == "" {
```

Comment:
@PombeirP what do you think about this? I am not sure why the fleet was required before. I think it's ok to define static nodes and bootnodes without a fleet for some cases. Or like in a case with canary node, define just static nodes.

Reply:
I think it's fine then. By the way, maybe the comment for `Enabled` in https://github.com/status-im/status-go/blob/default/params/config.go#L107 could be improved to define what it does, like `Enabled flag specifies that nodes in this configuration are taken into account`, and we should replace `for this fleet` in comments with `for this cluster`?